include/modules/github.hpp

```diff
@@ -18,12 +18,13 @@ namespace modules {
     bool build(builder* builder, const string& tag) const;
 
    private:
-    void update_label(const int);
+    void update_label(int);
```

Comment:
Is this from clang-format?

Reply:
clang-tidy rather than clang-format but yes it's from a clang tool ;)